<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserBackupDialog</class>
 <widget class="QDialog" name="UserBackupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>不清楚请留空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSchoolName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>不清楚请留空</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>学校名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>班级编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>密保答案：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leResult">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>不清楚请留空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePhone">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>不清楚请留空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturnPassword">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>210</y>
     <width>211</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
	font: 12pt &quot;黑体&quot;;
	background-color: rgb(0, 163, 255);
    border-radius:3px;
    color:white;
}
QPushButton:hover{
	background-color: rgb(60, 182, 251);
}</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leClassNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>不清楚请留空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
